README.md

```diff
@@ -18,10 +18,7 @@ to improve your code and the structure of your project!
 When you are ready send a pull-request that
 
 - Adds information about your project at the end of `data.yml`. The file
-  contains comments that will help you fill in the information; and
-
-- Adds an image or videos of your project, if not using URLs, to the `assets`
-  folder.
+  contains comments that will help you fill in the information.
```

Comment:
Did you mean to update the README?

Reply:
Yes since I don't see any `assets` folder in the repo, I thought that it might be a vestigial comment from how submissions used to work, so I "janitorialized" it... but perhaps I'm missing something? I just did not understand that comment in the README.md.

Reply:
In other words: the `assets` folder is perhaps auto-generated? A new user crafting a new submission will not see it right after cloning the repo as it's implied in the readme?

Reply:
... happy to rollback that particular change if it's blocking the review and merge process though :-S